.github/workflows/hlint.yml

```diff
@@ -21,4 +21,5 @@ jobs:
       uses: rwe/actions-hlint-run@v1
       with:
         hlint-bin: "hlint --with-group=extra --hint=ghcide/.hlint.yaml"
-        path: '[ "ghcide/src", "ghcide/exe", "ghcide/bench/lib", "ghcide/bench/exe", "ghcide/bench/hist", "shake-bench/src", "ghcide/test/exe" ]'
+        path: '[ "ghcide/src", "ghcide/exe", "ghcide/bench/lib", "ghcide/bench/exe", "ghcide/bench/hist", "shake-bench/src", "ghcide/test/exe" pop]'
```

Comment:
`"ghcide/test/exe" pop` seems to be put as a typo?

Reply:
oh yeah, good catch
```suggestion
        path: '[ "ghcide/src", "ghcide/exe", "ghcide/bench/lib", "ghcide/bench/exe", "ghcide/bench/hist", "shake-bench/src", "ghcide/test/exe"]'
```